tests/executables_src/testDeviceInitialisationHandler.cc

```diff
@@ -202,62 +211,62 @@ namespace Tests::testDeviceInitialisationHandler {
     // wait until the device is reported to be OK again (check with timeout),
     // then check the initialisation (again, no extra timeout needed because of the logic:
     // success is only reported after successful init).
-    CHECK_EQUAL_TIMEOUT(test.readScalar<int32_t>(ctk::RegisterPath("/Devices") /
-                            ctk::Utilities::escapeName(deviceCDD.data(), false) / "status"),
+    CHECK_EQUAL_TIMEOUT(test.readScalar<int32_t>(
+                            ctk::RegisterPath("/Devices") / ctk::Utilities::escapeName(deviceCDD, false) / "status"),
         0, 30000);
     // We use the macro here for convenience, it's a test, speed should not matter
     // NOLINTNEXTLINE(readability-container-size-empty)
     CHECK_EQUAL_TIMEOUT(test.readScalar<std::string>(ctk::RegisterPath("/Devices") /
-                            ctk::Utilities::escapeName(deviceCDD.data(), false) / "status_message"),
+                            ctk::Utilities::escapeName(deviceCDD, false) / "status_message"),
         "", 30000);
 
     // initialisation should be correct now
     BOOST_CHECK_EQUAL(var1, 42);
     BOOST_CHECK_EQUAL(var2, 47);
     BOOST_CHECK_EQUAL(var3, 52);
 
-    // now check that the initialisation error is also reportet when recovering
+    // now check that the initialisation error is also reported when recovering
     // Prepare registers to be initialised
     var1 = 12;
     var2 = 13;
     var3 = 14;
 
     // Make initialisation fail when executed, and then cause an error condition
     throwInInitialisation = true;
-    auto dummyBackend = boost::dynamic_pointer_cast<ctk::ExceptionDummy>(
-        ctk::BackendFactory::getInstance().createBackend(deviceCDD.data()));
+    auto dummyBackend =
+        boost::dynamic_pointer_cast<ctk::ExceptionDummy>(ctk::BackendFactory::getInstance().createBackend(deviceCDD));
     dummyBackend->throwExceptionWrite = true;
 
     auto reg4_cs = test.getScalar<int32_t>("/REG4");
     reg4_cs = 20;
     reg4_cs.write();
 
-    CHECK_EQUAL_TIMEOUT(test.readScalar<int32_t>(ctk::RegisterPath("/Devices") /
-                            ctk::Utilities::escapeName(deviceCDD.data(), false) / "status"),
+    CHECK_EQUAL_TIMEOUT(test.readScalar<int32_t>(
+                            ctk::RegisterPath("/Devices") / ctk::Utilities::escapeName(deviceCDD, false) / "status"),
         1, 30000);
     // First we see the message from the failing write
-    BOOST_CHECK(!test
-            .readScalar<std::string>(
-                ctk::RegisterPath("/Devices") / ctk::Utilities::escapeName(deviceCDD.data(), false) / "status_message")
-            .empty());
+    CHECK_EQUAL_TIMEOUT(test.readScalar<std::string>(ctk::RegisterPath("/Devices") /
```

Comment:
A CHECK_TIMEOUT would do here. The CHECK_EQUAL_TIMEOUT is used to print the two values in case the test does not pass. The message "true" is not "false" does not provide any added value.